serveradmin/apps/management/commands/createapp.py

```diff
@@ -0,0 +1,60 @@
+"""Serveradmin
+
+Create application for user
+
+Copyright (c) 2021 InnoGames GmbH
+"""
+from os import environ
+
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand, CommandError
+
+from serveradmin.apps.models import Application
+
+
+class Command(BaseCommand):
+    help = 'Create application for user'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--owner', help='Username of application owner')
+        parser.add_argument('--superuser', action='store_true',
+                            help='Create superuser token')
+        parser.add_argument('--non-interactive', action='store_true', help=(
+            'Gather values from SERVERADMIN_TOKEN_OWNER, SERVERADMIN_TOKEN '
+            'and SERVERADMIN_TOKEN_SUPERUSER env variables. '
+            'Any value for SERVERADMIN_TOKEN_SUPERUSER means yes. '
+            'Empty string or absence means no.'
```

Comment:
`None`, `empty list`, everything falseable means no. ^^

Reply:
There is no None, empty list etc. in shell if you define a variable your can write `SERVERADMIN_TOKEN_SUPERUSER=yes` or `SERVERADMIN_TOKEN_SUPERUSER=` or not define it. First one will always evaluate to true because it is a string > 0 later two will mark absence and therefore evaluate to False.

Reply:
True that.